Rakefile

```diff
@@ -177,13 +177,14 @@ if USE_RAKE_COMPILER
 
   ENV['RUBY_CC_VERSION'] ||= '1.8.7:1.9.3:2.0.0:2.1.5:2.2.1'
 
+  # To reduce the gem file size strip mingw32 dlls before packaging
   ENV['RUBY_CC_VERSION'].to_s.split(':').each do |ruby_version|
-    task "copy:ffi_c:i386-mingw32:#{ruby_version}" do |t|
```

Comment:
No, it shouldn't. It's an in-place operation, with no proper timestamp to compare.